conf/st2.dev.conf

```diff
@@ -64,7 +64,7 @@ port = 6001
 url = /webhooks/generic/
 
 [action_sensor]
-triggers_base_url = http://localhost:9101/triggertypes/
+triggers_base_url = http://localhost:9101/v1/triggertypes/
```

Comment:
I don't know if it was an intentional choice to not use the versioned URL here. Let me know and I can revert this change. Also should there be a trailing slash?